about.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+  <link rel="stylesheet" href="./styles/reset.css">
+  <link rel="stylesheet" href="./styles/about.css">
+</head>
+
+<body>
+  <header>
+    <section class="nave-header">
+      <div class="upper-nav">
+        <ul>
+          <li><i class="fa-brands fa-facebook-f"></i></li>
+          <li><i class="fa-brands fa-twitter"></i></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="desktop-menu">
+        <div id="desktop-menu__logo">
+          <a href="./index.html"> <img src="./assets/logo.png" alt="" srcset=""> </a>
+        </div>
+        <nav class="desktop-menu__links">
+          <ul class="desktop-menu__links_list">
+            <li><a href="./about.html">About</a></li>
```

Comment:
[Optional] I suggest that you add a navigation back to the home page from the about page. This will make for a better user experience.

![image](https://github.com/majdalkilany/Software-Developer-Summit/assets/20554583/f5c2375a-1ce9-442d-8ef7-d707ce4671ab)